test/unit/HTMLBuilder.test.ts

```diff
@@ -165,6 +165,17 @@ describe('HTMLBuilder', () => {
     expect(res.includes(falseStyleString)).toBeTruthy();
   });
 
+  test('statement-level HTML is added to statement results when calculation option is specified', () => {
+    const res = generateHTML(simpleMeasure, [elm], statementResults, trueClauseResults, 'test', {
+      buildStatementLevelHTML: true
+    });
+
+    expect(statementResults[0].statementLevelHTML).toBeDefined();
```

Comment:
The only other case I can think of for now is maybe one that makes sure there is no `statementLevelHTML` when the relevance is NA?